scripts/build.d/python

```diff
@@ -64,15 +64,18 @@ pushd ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src/${pkgfull} > /dev/null
   cfgcmd+=("--prefix=${PREFIX}")
   cfgcmd+=("--enable-shared")
   cfgcmd+=("--enable-ipv6")
-  cfgcmd+=("--with-ensurepip=no")
   cfgcmd+=("--enable-optimizations")
+  cfgcmd+=("--without-ensurepip")
+  cfgcmd+=("--with-system-expat")
+  cfgcmd+=("--with-system-ffi")
+  cfgcmd+=("--with-system-libmpdec")
+  cfgcmd+=("--with-readline=editline")
   cfgcmd+=("--with-lto")
-  cfgcmd+=("--with-tcltk-includes=-I${PREFIX}/include")
```

Comment:
On my mac, the tcltk arguments are not effective so I remove them.  If they are useful on other platform, please add them back.